.github/workflows/ci.yml

```diff
@@ -53,5 +53,4 @@ jobs:
         flags: unittests
         env_vars: OS,PYTHON
         name: codecov-umbrella
```

Comment:
I think we should be able to roll back this change now

Reply:
That seems okay to me - the coverage checks are working fine from the PR target side, we don't need them twice.